mythril/laser/ethereum/svm.py

```diff
@@ -180,6 +171,22 @@ def sym_exec(
             )
             log.info("Achieved {:.2f}% coverage for code: {}".format(cov, code))
 
+    def sym_exec(
+        self, main_address=None, creation_code=None, contract_name=None, timeout=None
+    ) -> None:
+        timeout = timeout or self.execution_timeout
+        with ThreadPoolExecutor(max_workers=1) as executor:
+            f_results = executor.submit(
+                self._sym_exec,
+                main_address=main_address,
+                creation_code=creation_code,
+                contract_name=contract_name,
+            )
+            try:
+                return f_results.result(timeout)
+            except TimeoutError:
+                return
```

Comment:
It might be nice to log something here so that it is clear that a timeout occurred? WDYT?